assets/css/client.css

```
@@ -1,401 +1,499 @@
+.btn-outline-secondary {
+    color: var(--bc-text-secondary);
+    border-color: var(--bc-text-secondary);
+}
+.btn-outline-secondary:hover {
+    background-color: var(--bc-text-secondary);
+    border-color: var(--bc-text-secondary);
+    color: var(--bc-white);
+}
+
+.btn-success { background-color: var(--bc-success); border-color: var(--bc-success); color: var(--bc-white); }
+.btn-success:hover { background-color: darken(var(--bc-success), 8%); border-color: darken(var(--bc-success), 8%); color: var(--bc-white); }
```

Biome (1.9.4)

[error] Unexpected unknown function: darken

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)